CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Versions
 
 ## [Unreleased]
-
+* Add additional coverage for SQL 2016 STIG
+* Implement new rule to execute PowerShell scripts
```

Comment:
Please remove the  * Add additional coverage for SQL 2016 STIG,
and update your issue note to : 
* Update PowerSTIG to add PowerShellScriptQuery Rule [#921](https://github.com/microsoft/PowerStig/issues/921)

Reply:
Per discussion : we are going to hold on this and Upgrade SQLServerDSC, then revisit.